lib/ConfigFileProcessor.js

```diff
@@ -13,11 +13,16 @@
  * contents is a violation of applicable laws.
  */
 
+const fs = require('fs');
 const async = require('async');
-
-const { ConfigType } = require('./Constants');
+const path = require('path');
+const logger = require('./logger.js');
```

Comment:
For better or worse, I've always envisioned that the `logger` module will be used through the `CLIManager`'s `log` method. 'Cause this is where the `logger` is 'initialized' depending on the options passed to the CLI by the user. And now I see that even there's additional logic in this `log` method. So I think that either some refactoring should occur or logging here should happen through the `log` method.

Reply:
Hmmm, I copied this from Utils.js where the logger is used directly. In general it is a good practice for the logger to be a signleton, e.g. have a configured instance that is accessible from anywhere.

Reply:
I'm not opposing the idea 😃 
The usage in Utils was a leftover meant to be changed. I don't think we should have logging in Utils functions, anyways.